README.md

```diff
@@ -42,38 +45,39 @@ To get started, follow the instructions outlined in [the documentation](https://
 ## 🎨 Features
 
 - **Basic syntax highlighting** using a complete and up-to-date [TextMate
-  grammar](https://macromates.com/manual/en/language_grammars). _This grammar is slated
-  to drive GitHub's syntax highlighting for WDL files [in a future
-  release](https://github.com/github-linguist/linguist/pull/6972)_.
-- **Document and workspace diagnostics** courtesy of the language server protocol
-  implementation provided by`sprocket analyzer`.
+  grammar](https://macromates.com/manual/en/language_grammars). _This grammar
+  is slated to drive GitHub's syntax highlighting for WDL files [in a future release](https://github.com/github-linguist/linguist/pull/6972)_.
```

Comment:
Any info on when this will actually hit production? The current syntax highlighting is really lousy. Just completely breaks on Bash heredocs which we use somewhat often.

Reply:
I don't know if GitHub publishes what version of linguist they're using in production, but the WDL changes made it into 8.0.0.